src/main/play-doc/intro/Clustering-Considerations.md

```diff
@@ -0,0 +1,42 @@
+# Typesafe ConductR %PLAY_VERSION%
+
+## Cluster Resiliency Considerations
+
+In order to achieve cluster resilience when using ConductR, it’s recommended for the cluster to be separated into at least 3 distinct parts, e.g.
+
```

Comment:
I can't think of better words than "3 distinct parts" - better suggestions are appreciated